addons/barcodes/static/src/barcode_service.js

```diff
@@ -39,7 +39,8 @@ export const barcodeService = {
         function handleBarcode(barcode, target) {
             bus.trigger('barcode_scanned', {barcode,target});
             if (target.getAttribute('barcode_events') === "true") {
-                $(target).trigger('barcode_scanned', barcode);
```

Comment:
This doesn't trigger the event properly so I had to replace.

Reply:
very good